app/src/main/java/ir/ac/shirazu/softwareproject/server_api/Meal/MyKit.java

```diff
@@ -24,10 +30,11 @@
     private static String loginURL = "http://nowaw.pythonanywhere.com/login/login";
     private static String studentInfoURL = "http://nowaw.pythonanywhere.com/login/self_data";
     private static String logoutURL = "http://nowaw.pythonanywhere.com/login/logout";
+    private static String deleteURL = "http://nowaw.pythonanywhere.com/login/delete";
     private static HttpURLConnection connection;
     private static String USER_AGENT = "Mozilla/5.0";
     private static String token;
```

Comment:
@hosseindehghanipour1998 
Is this token different from Student.user_token?

Reply:
Nope they are both the same but I couldn't reach the token sent by the server in IONIC function so 
 I had to define a static token. I will inform you later on about this specific variable.